js/todo.js

```diff
@@ -0,0 +1,55 @@
+var myNodelist = document.getElementsByTagName("LI");
+var i;
+for (i = 0; i < myNodelist.length; i++) {
+    var span = document.createElement("SPAN");
+    var txt = document.createTextNode("\u00D7");
+    span.className = "close";
+    span.appendChild(txt);
+    myNodelist[i].appendChild(span);
+}
+
+// Click on a close button to hide the current list item
+var close = document.getElementsByClassName("close");
+var i;
+for (i = 0; i < close.length; i++) {
+    close[i].onclick = function() {
+        var div = this.parentElement;
+        div.style.display = "none";
+    }
+}
+
+// Add a "checked" symbol when clicking on a list item
+var list = document.querySelector('ul');
```

Comment:
Musisz zwracać uwagę kiedy się to wykona.
dołączając skrypt na na początku pliku html, ten kawałek kody możne wykonać się przed utworzeniem elementów strony.

dla tego w jQuery jest metoda on load - poszukają odpowiednika "natywnego". (bez jQuery)

po zatym umiesc wszytko w fukcji np. function main() { }